### ISSUE TITLE
Application Form

### ISSUE THREAD

**Issue by hanarimawi** [hack4impact-uiuc/recruitment-portal-product-infra#4]
Within a Season, Admins can edit rounds and specify a season’s schedule
1. There will always be an Application round within a season, they can add more rounds
2. This is a google form. Where does that information go? How can we upload it to the db smoothly?
3. When an admin clicks the Application form, they are directed to a page where they link the google form they create outside of the app
4. We take this link and embed the google form inside our app

How to create a candidate profile from this information
In short, Parsing google forms
